ecrecover/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+
+[package]
+name = "ewasm-precompile-ecrecover"
+version = "0.1.0"
+authors = ["Guillaume Ballet <gballet@gmail.com>"]
+edition = "2018"
```

Comment:
Are we requiring rust 2018?

Reply:
It's a new piece of code, might as well be up to standards